docs/docs/dev_docs/contracts/deploying.md

```diff
@@ -27,35 +59,101 @@ Set this only if this contract is expected to receive private notes (in such a c
 By default is set to a random value.
 Set it, if you need a deterministic contract address (same functionality as Ethereum's `CREATE2` opcode).
 
-To give you a more complete example we will deploy the `PrivateToken` contract whose artifacts are included in the `@aztec/noir-contracts` package.
+</TabItem>
+<TabItem value="js" label="Aztec.js">
+
+The `deploy()` method is generated automatically with the typescript class representing your contract. It requires the RPC to be passed (this can be the `AztecRPC` client) and any arguments to be passed to the constructor.
+
+Additionally the `.send()` method can has a few optional arguments too, which are specified in an optional object:
+- `portalContract?: EthAddress`: The L1 portal address to link the contract to. See the section on [Portals to learn more about them](./portals/main.md)
+- `contractAddressSalt?: Fr`: A salt that can be passed which is used to compute the contract address of the contract being deployed. 
```

Comment:
```suggestion
- `contractAddressSalt?: Fr`: A salt which is one of the inputs when computing a contract address of the contract to be deployed. 
```

Reformulated this a bit because it sounded a bit as if the salt was the only input.